test/consensus_cogateway/confirm_open_kernel.js

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const { AccountProvider } = require('../test_lib/utils.js');
+const DeclareOpenKernel = require('./declare_open_kernel.json');
+
+const SpyCoconsensus = artifacts.require('SpyCoconsensus');
+
+const ConsensusCoGateway = artifacts.require('ConsensusCogatewayTest');
+const SpyAnchor = artifacts.require('SpyAnchor');
+
+contract('CoConsensusGateway::confirmOpenKernel', (accounts) => {
```

Comment:
```suggestion
contract('CoconsensusGateway::confirmOpenKernel', (accounts) => {
```
correct at other places too.

Reply:
Done